configuration/jsr223.md

```diff
@@ -335,6 +337,47 @@ With `scriptExtension.get("automationManager")` the `automationManager` can be m
 - `events.storeStates(Item...)`
 - `events.restoreStates(Map<Item, State>)`
 
+#### `lifecycle` Preset (`importPreset` not required)
+
+It provides a mechanism to execute code, when the input file is deleted.  Modifying a file deletes it and creates it again.
```

Comment:
2nd sentence has be in the next line. (make sure the first sentence doesn't have a trailing space then.